chart/templates/ingress.yaml

```diff
@@ -37,7 +37,8 @@ spec:
   - host: {{ .Values.hostname }}  # hostname to access rancher server
     http:
       paths:
-      - backend:
+      - pathType: ImplementationSpecific
```

Comment:
I think this field is new in the networking.k8s api, so should we add a check for the api version?
`{{- if .Capabilities.APIVersions.Has "networking.k8s.io/v1/Ingress" }}`